Packs/Core/Playbooks/playbook-A_mail_forwarding_rule_was_configured_in_Google_Workspace.yml

```diff
@@ -0,0 +1,1784 @@
+    type: condition
+    task:
+      id: de8e5625-536a-4bb9-8bdc-70ee14eb72ff
+      version: -1
+      name: Check if the forwarding mail address still exists
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "17"
+      "yes":
+      - "25"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Gmail.ForwardingAddress.forwardingEmail
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 710,
+          "y": 1700
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: fe775a32-55a3-45a0-8502-c6e319e7ae91
+    type: title
+    task:
+      id: fe775a32-55a3-45a0-8502-c6e319e7ae91
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 440,
+          "y": 2580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "21":
+    id: "21"
+    taskid: 252d8473-0602-420c-8fef-df880efcc695
+    type: regular
+    task:
+      id: 252d8473-0602-420c-8fef-df880efcc695
+      version: -1
+      name: Check if the rule was created outside of working hours
+      description: Checks whether the given value is within the specified time (hour) range.
+      scriptName: BetweenHours
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "45"
+    scriptarguments:
+      begin_time:
+        simple: "22:00:00"
+      end_time:
+        simple: "06:00:00"
+      extend-context:
+        simple: IsOutOfWorkingHours=
+      value:
+        complex:
+          root: Core.OriginalAlert.raw_abioc.event
+          accessor: event_timestamp
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 765
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: a1b2240d-c96c-46a2-8749-c94f8a214538
+    type: regular
+    task:
+      id: a1b2240d-c96c-46a2-8749-c94f8a214538
+      version: -1
+      name: Check if the user connected from an unusual geolocation
```

Comment:
Change to "Check for unusual geolocation connections" it better describes what the task is doing